microsoft/electionguard-web-api
# ⚙️ Host as A Windows Service

Fixes [Host as A Windows Service](https://github.com/microsoft/ElectionGuard-SDK-DotNetCore-Reference-Web-API/issues/8)

## To Test

on a windows machine, from an elevated command prompt:
1. `pwsh .\scripts\Create-Windows-Service.ps1`
2. open `services.msc` and enable the `ElectionGuardService`
3. from a browser (or postman) `http://localhost:5001/election`
4. Expected: `Welcome to ElectionGuard` is returned

Merge after #10
